src/plugins/data_source_management/public/components/edit_data_source/components/edit_form/__snapshots__/edit_data_source_form.test.tsx.snap

```diff
@@ -520,177 +521,82 @@ exports[`Datasource Management: Edit Datasource Form Case 1: With Username & Pas
                 className="euiHorizontalRule euiHorizontalRule--full euiHorizontalRule--marginMedium"
               />
             </EuiHorizontalRule>
-            <EuiDescribedFormGroup
-              description={
-                <p>
-                  <FormattedMessage
-                    defaultMessage="This connection information is used for reference in tables and when adding to a data source connection"
-                    id="dataSourcesManagement.editDataSource.endpointDescription"
-                    values={Object {}}
-                  />
-                </p>
-              }
-              title={
-                <h4>
-                  <FormattedMessage
-                    defaultMessage="Endpoint"
-                    id="dataSourcesManagement.editDataSource.endpointTitle"
-                    values={Object {}}
-                  />
-                </h4>
-              }
+            <EuiFormRow
```

Comment:
Can we remove the snapshot tests for components like this? This diff does not make sense for the corresponding change in the test file and is a symptom of a bad snapshot. If we want to test the functionality of the component or its UI, we should leverage the functional test repo. Unit tests should only test the logic. the same comment applies to `edit_data_source`.

> You need to test your components as a unit and in isolation, if one react component output changes only one unit test fails.

This principle breaks here because if EUI changes the markup for this component, all these tests will break.